src/pre_defined_classes/mathematical_program/hard/q5/MainTest.java

```diff
@@ -0,0 +1,31 @@
+package pre_defined_classes.mathematical_program.hard.q5;
```

Comment:
Oh wait why is this file here?

Reply:
I have no idea. I think something happened while I was messing around with this file while waiting to be able to use negatives and it pushed it when I flipped over to working on this program.